analysis/config/config_test.go

```diff
@@ -226,7 +227,8 @@ func TestLoadVersionBefore_v0_3_0_Errors(t *testing.T) {
 
 func TestLoadWithReports(t *testing.T) {
 	c := NewDefault()
-	c.ReportsDir = "example-report"
+	wd, _ := os.Getwd()
+	c.ReportsDir = path.Join(wd, "testdata/example-report")
```

Comment:
I think `filepath.Join` is preferred over `path.Join` for better OS compatibility: https://pkg.go.dev/path/filepath#Join